### PR TITLE
fix: tag value handling

### PR DESCRIPTION
**Pull request by Friendseeker**
### Issue

`_getTagValue` has following problems

- Case 9 (SLONG), Case 129 (UTF-8) is not handled
- Case 10 is as quotient of two SLONGs, yet currently it is fetched as quotient of two LONGs
- All of Case 1,3,4,5,9,10 can result in array of values, yet only case 5 returns an array

This PR fixes the above 3 problems, so `_getTagValue` is fully in sync with EXIF spec.

### CODE REVIEW

File: src/markers/utils.js [hunk 1]
```diff
@@ -53,3 +53,11 @@ export const concatenateUint8Arrays = (arrays) => {
...
   return concatenatedArray;
 };
+
+export const readInt32BE = (array, offset) => {
+  return readUInt32BE(array, offset) | 0;
+};
+
+export const readInt32LE = (array, offset) => {
+  return readUInt32LE(array, offset) | 0;
+};
```
Comment (diegomura) [inline src/markers/utils.js]: Why are these needed?

Also please add a new line at the end of the file :)

Comment (Friendseeker) [reply src/markers/utils.js]: These are for handling Case 9 & 10. Here's excerpt from EXIF standard:

> 9 = SLONG A 32-bit (4-byte) signed integer (2's complement notation).
10 = SRATIONAL Two SLONGs. The first SLONG is the numerator and the second SLONG is the
denominator.

Hence I added these methods to read signed ints, as existing methods only read unsigned numbers.
